commands/cloudapi-v6/lan.go

```diff
@@ -188,6 +193,7 @@ Required values to run command:
 	update.AddBoolFlag(constants.ArgWaitForRequest, constants.ArgWaitForRequestShort, constants.DefaultWait, "Wait for the Request for LAN update to be executed")
 	update.AddIntFlag(constants.ArgTimeout, constants.ArgTimeoutShort, constants.DefaultTimeoutSeconds, "Timeout option for Request for LAN update [seconds]")
 	update.AddInt32Flag(cloudapiv6.ArgDepth, cloudapiv6.ArgDepthShort, cloudapiv6.DefaultUpdateDepth, cloudapiv6.ArgDepthDescription)
+	update.AddStringFlag(cloudapiv6.FlagIPv6CidrBlock, "", "disable", fmt.Sprintf(cloudapiv6.FlagIPv6CidrBlockDescription, 64, "Datacenter"))
```

Comment:
`fmt.Sprintf(cloudapiv6.FlagIPv6CidrBlockDescription, 64, "Datacenter"))`

Dealing with placeholders for constants is unreadable. Could you maybe 

1. deal with the placeholders inside of the constants pkg (make it a `var` declaration), or
2. remove it as a constant and deal with the placeholders here?